src/main/java/frc/robot/joystickManager/JoystickManager.java

```diff
@@ -0,0 +1,136 @@
+package frc.robot.joystickManager;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.Robot;
+import frc.joysticks.JoystickPorts;
+import frc.joysticks.SmartJoystick;
+
+import frc.joysticks.JoystickPorts.*;
+
+public class JoystickManager {
+
+	private SmartJoystick mainJoystick;
+	private SmartJoystick secondJoystick;
+	private SmartJoystick thirdJoystick;
+	private SmartJoystick fourthJoystick;
+	private SmartJoystick fifthJoystick;
+	private SmartJoystick sixthJoystick;
+
+	private SmartJoystick[] joysticks;
+
+	private final Robot robot;
+
+	public JoystickManager(Robot robot) {
+		joysticks = new SmartJoystick[6];
+
+		int currentPort = 0;
+
+		for (SmartJoystick joystick : joysticks) {
+			SendableChooser<JoystickState> stateChooset = new SendableChooser<>();
+			addOptions(stateChooset, intToPort(currentPort));
+			currentPort++;
+		}
+
+		this.robot = robot;
+	}
+
+	public void setJoystickState(JoystickPorts port, JoystickState joystickState) {
+		if (joystickState == JoystickState.NONE && portToJoystick(port) != null) {
+			portToJoystick(port).setState(joystickState);
+		} else if (portToJoystick(port) == null) {
+			switch (port) {
+				case MAIN:
+					mainJoystick = new SmartJoystick(port, joystickState);
+					JoystickBindings.configureBindings(mainJoystick, robot);
+					break;
+
+				case SECOND:
+					secondJoystick = new SmartJoystick(port, joystickState);
+					JoystickBindings.configureBindings(secondJoystick, robot);
+					break;
+
+				case THIRD:
+					thirdJoystick = new SmartJoystick(port, joystickState);
+					JoystickBindings.configureBindings(thirdJoystick, robot);
+					break;
+
+				case FOURTH:
+					fourthJoystick = new SmartJoystick(port, joystickState);
+					JoystickBindings.configureBindings(fourthJoystick, robot);
+					break;
+
+				case FIFTH:
+					fifthJoystick = new SmartJoystick(port, joystickState);
+					JoystickBindings.configureBindings(fifthJoystick, robot);
+					break;
+
+				case SIXTH:
+					sixthJoystick = new SmartJoystick(port, joystickState);
+					JoystickBindings.configureBindings(sixthJoystick, robot);
+					break;
```

Comment:
perhaps change the port to be an int and save all the code duplication by just using the joysticks array